### ISSUE TITLE
Error in ligand_alchemy.py

### ISSUE THREAD

**Issue by qqq12a** [deGrootLab/pmx#24]
```
b'  File "/home/---/miniconda3/envs/AZtutorial/lib/python3.7/site-packages/pmx/ligand_alchemy.py", line 774, in _isTriple'
b'    if(getBondLength(mol,a.GetIdx(),neighb.GetIdx())<1.25 ): # need to check bond length (in Angstroms)'
```

Maybe it shouldn't be  **getBondLength**, but **self._getBondLength**

**Comment by vgapsys** [comment]
Thanks! Fixed in the commit 0e58425a08c3031774164b201eb7530e2a83ae35

**Comment by qqq12a** [comment]
Thanks for the nice tool.
